### ISSUE TITLE
mpm help

### ISSUE THREAD

**Issue by bendichter** [mobeets/mpm#56]
What do you think of `>> mpm` with no args bringing up a help/man/doc/usage-guide?

**Comment by mobeets** [comment]
Yeah, that's a good idea. I should also update the help/doc info, to make it more like the README
